recipes/plotly/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "1.12.4" %}
+{% set sha256 = "6136ebcc6cd6435b5d238385b395d856e75773a58c303c5107c9834ca5577bbd" %}
+
+package:
+  name: plotly
+  version: {{ version }}
+
+source:
+  fn: plotly-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/plotly/plotly-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - requests
+    - six
+    - pytz
```

Comment:
Are you sure that `requests`, `six`, `pytz` are not `run` time dependencies? (Maybe only `six` is also `build` time.)